assets/model_monitoring/components/src/model_monitor_data_joiner/run.py

```diff
@@ -83,6 +83,11 @@ def run():
         args.right_join_column
     )
 
+    # Check if there are any duplicate column names
+    columns = joined_data_df.columns
+    if len(columns) != len(set(columns)):
+        raise Exception(f"Duplicate column names found: {columns}")
```

Comment:
do we want to print all the columns but not only the duplicated columns?

Reply:
good point will change statement